test/distributed/org/apache/cassandra/distributed/impl/Coordinator.java

```diff
@@ -63,7 +63,14 @@ public Coordinator(Instance instance)
     @Override
     public SimpleQueryResult executeWithResult(String query, ConsistencyLevel consistencyLevel, Object... boundValues)
     {
-        return instance().sync(() -> unsafeExecuteInternal(query, consistencyLevel, boundValues)).call();
```

Comment:
in this case I think we should revert, but think we should change this behavior on trunk... I strongly regret making the decision to not "wrap" the internal exceptions... its so much harder to debug =(

Reply:
ok